projects/libreoffice/build.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash -eu
 
+# afl++ workaround, this also randomly enables CMPLOG lite
+test -n "$AFL_MAP_SIZE" && {
+  export CC=clang
+  export CXX=clang++
+  AFL_FLAGS="-fsanitize-coverage=trace-pc-guard"
+  rm -f $WORK/afl_cmplog
+  # 40% chance for cmplog lite
+  test $(($RANDOM % 10)) -lt 4 && {
```

Comment:
since we are trying this in one test project, can remove the random part and do cmplog 100% of time.

Reply:
for a test maybe a different project? maybe putting that in afl_compile and matching on the target name?
I will disable this here then for the time being.